src/pybind/mgr/dashboard/frontend/src/app/shared/services/notification.service.ts

```diff
@@ -63,18 +65,45 @@ export class NotificationService {
     localStorage.setItem(this.KEY, JSON.stringify(recent));
   }
 
+  queueNotifications(notifications: CdNotificationConfig[], ms: number = 500) {
```

Comment:
Shouldn't we try to gather all timeouts and use a global constant/variable vs. using per-callback literals?

Reply:
Do you mean like a class where all timeout times in use are stored? IMHO this is something for a different PR.